src/Annotation/Controller/Negotiations.php

```diff
@@ -12,7 +11,6 @@
  * @Target("METHOD")
  * @NamedArgumentConstructor()
  */
-#[Attribute(Attribute::TARGET_METHOD)]
 final class Negotiations
```

Comment:
Nested attributes are not allowed, so argument has to be used in a standalone way, e.g.:

```diff
-/**
- * @Negotiations([
- *     @Negotiation(...)
- *     @Negotiation(...)
- * ])
+#[Negotiation(...)]
+#[Negotiation(...)]
```

See https://wiki.php.net/rfc/attributes_v2#why_are_nested_attributes_not_allowed